Sortable.js

```diff
@@ -683,11 +683,31 @@
 						}
 
 						if (!dragEl.contains(el)) {
-							if (after && !nextSibling) {
-								el.appendChild(dragEl);
-							} else {
-								target.parentNode.insertBefore(dragEl, after ? nextSibling : target);
-							}
+							swapElements(dragEl, target);
```

Comment:
This code brakes 'after' logic:
 - https://github.com/RubaXa/Sortable/blob/dev/Sortable.js#L733-L735
 - https://github.com/RubaXa/Sortable/blob/dev/Sortable.js#L740-L750